src/main/java/org/octri/hpoonfhir/view/ObservationModel.java

```diff
@@ -17,34 +19,65 @@
 	private final String fhirId;
 	private final String loincId;
 	private final String description;
-	private final String date;
+	private final String startDate;
+	private final String endDate;
 	private final String value;
 
 	public ObservationModel(String loincId, ObservationLoincInfo observationLoincInfo) {
+		DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
 		this.fhirId = observationLoincInfo.getFhirId();
 		this.loincId = loincId;
 		this.description = observationLoincInfo.getDescription();
-		this.date = observationLoincInfo.getDate();
+		this.startDate = observationLoincInfo.getStartDate().map(d -> df.format(d)).orElse("");
```

Comment:
This is a neat use of `Optional.map`. I had to look in your fhir2hpo branch to see what `startDate` was. If this formatting was performed in more places I might suggest extracting a `Function`.